config/routes.rb

```diff
@@ -2,6 +2,8 @@
 
 
   get '/products' => 'products#index'
+  get '/products/new'=> 'products#new'
```

Comment:
to make these a little more readable, try the syntax:
`get '/products/new', to: 'products#new'`